go/pkg/client/bytestream.go

```diff
@@ -83,7 +83,7 @@ func (c *Client) ReadResourceToFile(ctx context.Context, name, fpath string) (in
 }
 
 func (c *Client) readToFile(ctx context.Context, name string, fpath string) (int64, error) {
-	f, err := os.Create(fpath)
+	f, err := os.OpenFile(fpath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, c.RegularMode)
```

Comment:
Can you also change this to `0666`? That was the previous behaviour with Create() - https://golang.org/pkg/os/#Create

Reply:
done